recipes/mopac/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mopac" %}
+{% set version = "22.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmopac/mopac/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 336db023275f39dd0d1b8d02f65b2db417c8aa56b539530d02639662e6e05d87
+  patches:
+    - mopac-bz.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('fortran') }}  # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+    - cmake
+    - ninja
+    - python
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+    - libblas
+    - liblapack
```

Comment:
Warning on OSX that [blas is not being linked dynamtically](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=528147&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=6155) to the conda-forge blas. Instead they are linking to some system [MacOS accelerate framework BLAS](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=528147&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1966). I'm not sure if that is preferred or not. This might cause problems for your users if this framework is not installed on ALL osx devices.

Reply:
At least on OSX/Arm64 the accelerate framework BLAS should give decent single core performance, not sure whether this is true for OSX/x86_64 there I think it was somewhat broken...

Adding `-DBLA_VENDOR=Generic` does the trick ([log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=528470&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1961)). This is supposed to be the default when building against the `libblas` / `liblapack` unless the implementation is explicitly used. Might be possible to export via the netlib version of the recipes, having it in `CMAKE_ARGS` would be cool, but I think this would be more difficult.